tests/src/test/scala/whisk/core/controller/test/ActivationsApiTests.scala

```diff
@@ -72,7 +75,7 @@ class ActivationsApiTests extends ControllerTestCommon with WhiskActivationsApi
         activations foreach { put(activationStore, _) }
         waitOnView(activationStore, namespace, 2)
         whisk.utils.retry {
-            Get(s"$collectionPath") ~> sealRoute(routes(creds)) ~> check {
+            Get(s"$collectionPath") ~> Route.seal(routes(creds)) ~> check {
```

Comment:
maybe to avoid all these changes in this file and others, can you import Route.seal as sealRoute?